Lib/ufo2ft/makeotfParts.py

```diff
@@ -180,6 +180,6 @@ def setupFile_featureTables(self):
                     self.outline[tag] = table
 
         elif self.features.strip():
-            feapath = os.path.join(self.font.path, "features.fea")
+            feapath = os.path.join(self.font.path if self.font.path is not None else '', "features.fea")
```

Comment:
If you don't have a valid path to the file then just leave `feapath` to None in that case. Relative imports will fail that's all.